api/server.go

```diff
@@ -691,6 +693,25 @@ func (s *server) pubkeyHostHandler(jc jape.Context) {
 	jc.Encode(hosts[0])
 }
 
+func (s *server) pubkeyHostScanHandler(jc jape.Context) {
+	// We could use jape.BasicAuth when defining the route in the map, but it
+	// makes the jape linter think that the route is undefined, so we have some
+	// auth code here.
+	if _, p, ok := jc.Request.BasicAuth(); !ok || s.manualScanPassword == "" || p != s.manualScanPassword {
```

Comment:
In `walletd` we define `wrapAuthHandler`, would that be a solution to the linting problem? Not sure I fully understand why the jape linter would think the route is undefined, never encountered that before myself.

Reply:
Ok. Took a quick look and I agree it is annoying. I tried splitting it in authenticated and unauthenticated routes to then merge the two routers, which works but doesn't solve the problem because jape loses track of all handlers 😁

```
authRoutes := auth(apiPassword)(jape.Mux(map[string]jape.Handler{
		"POST   /hosts/:key/scan":        srv.pubkeyHostScanHandler,
		"POST   /syncer/broadcast/block": srv.syncerBroadcastBlockHandler,
	}))
```